addons/l10n_mx/__manifest__.py

```diff
@@ -44,5 +44,6 @@
         "data/account_chart_template_data.xml",
         "views/res_config_settings_views.xml",
         "views/account_views.xml",
+        "data/account_types.xml",
```

Comment:
Hello @hbto 

I get this error:
![Screen Shot 2019-08-15 at 11 46 51 AM](https://user-images.githubusercontent.com/7606656/63111348-1a8ab700-bf53-11e9-9f70-4c58238815d0.png)

Maybe because my instance already exists, but could you create a dummy PR for enterprise-mx, please?

Regards

Reply:
No I think the problem is that the definition shall come first than the implementation.

![Screen Shot 2019-08-15 at 2 15 52 PM](https://user-images.githubusercontent.com/7598010/63120152-5760a900-bf67-11e9-802d-d426e97445c9.png)

I will fix it here

Reply:
@luistorresm dummy PR done at https://git.vauxoo.com/vauxoo/enterprise-mx/merge_requests/611/

<img width="744" alt="Screen Shot 2019-08-15 at 4 36 32 PM" src="https://user-images.githubusercontent.com/7598010/63128569-ddd2b600-bf7a-11e9-8887-3af3b346cb51.png">